src/Netflix/Netflix.ts

```diff
@@ -234,82 +268,88 @@ export default <IPluginBase & IPlugin>{
     version: "1.0.0",
     init: () => contextManager.enable(),
     destroy: () => contextManager.disable(),
+    homophones: {
+      /**
+       * Override Google plugin's search homophones
+       */
+      search: "search"
```

Comment:
Can you mention this workaround in the top-level comment - so we know to remove it once that issue is fixed in platform-space

Reply:
Sure